docs/chapters/baseplatform/deploying-the-image.rst

```diff
@@ -3,5 +3,5 @@
 
 Image names for PELUX targets
 -----------------------------
-* NUC/Minnowboard: ``core-image-pelux-intel-corei7-64.wic``
+* NUC, Minnowboard and ARP with intel SMARC: ``core-image-pelux-intel-corei7-64.wic``
```

Comment:
Intel. Do we really support Minnowboard?

Reply:
I do not know but I do not want to change that in this commit

Reply:
Intel